packages/design-system/src/Radio/Radio.styles.tsx

```diff
@@ -1,9 +1,16 @@
 import styled, { css } from "styled-components";
 
 const Variables = css`
+  --ads-v2-colors-control-radio-tack-default-bg: var(--ads-v2-color-bg);
```

Comment:
I don't think it's `tack`; after our conversation I looked this up and then I went and changed the category token naming - can you take a look at the history of edits on the page to confirm? I think this should be `dot`, but not sure.

Reply:
Done